aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -87,7 +87,10 @@ You need to create the database, and you'll use the EF Core [Migrations](xref:da
 
 [!code-xml[Main](start-mvc/sample/MvcMovie/MvcMovie.csproj?range=22-25&highlight=3)] 
 
-Note: The version numbers shown above were correct at the time of writing.
+Notes: 
+
+* You can't add the EF tools packages to the `DotNetCliToolReference` element using the Package Manager Console.
```

Comment:
I think you may want to update the document to add `Microsoft.EntityFrameworkCore.Tools` instead of `Microsoft.EntityFrameworkCore.Tools.DotNet`. The former works with .NET Core projects in VS better, and can be added using the Package Manager Console. The later is designed for `dotnet.exe` scenarios only, not VS.

cc @divega